Website/src/rendering/src/assets/css/components/Checkout/CartDetails.css

```diff
@@ -0,0 +1,197 @@
+.cart-details {
+  h1 {
+    @apply text-2xl font-bold my-3;
+  }
+
+  &-grid {
+    @apply flex flex-col-reverse lg:flex-row;
+  }
+
+  &-items {
+    @apply w-full lg:mr-4;
+  }
+
+  &-actions {
+    @apply mb-3 lg:w-1/3 lg:max-w-xs lg:relative lg:mb-0;
+
+    &-wrapper {
+      @apply lg:sticky lg:top-4;
+
+      > * {
+        @apply w-full;
+
+        &:not(:last-child) {
+          @apply mb-3;
+        }
+      }
+    }
+  }
+}
+
+.cart-summary {
+  @apply px-6 bg-white-dark pb-1;
+
+  strong {
+    @apply text-xl;
+  }
+}
+
+.quantity-input {
```

Comment:
@NellyKostadinova, Having `.quantity-input` as a top-level class in this file will affect all elements with that class in all the components of the solution. Same for all the classes defined below.

If we expect other top-level components to have the same `QuantityInput` sub-component with the same style, it would be best to extract the `.quantity-input` CSS class in its own `QuantityInput.css` file.

On the other hand, if the `QuantityInput` sub-component will only ever be used in this top-level `CartDetails` component, then it would be best to nest its CSS class inside `.cart-details`.

Same applies to other top-level CSS classes defined in this file.